meta-ros-common/recipes-extended/geographiclib/geographiclib_1.48.bb

```diff
@@ -11,4 +11,23 @@ SRC_URI[sha256sum] = "7203d56123b6f6fb31842295d57b2418f79fb0db9a06f2f65ee9e415c6
 
 S = "${WORKDIR}/GeographicLib-${PV}"
 
+PACKAGES += "${PN}-tools"
```

Comment:
This won't work, the packages are processed in the order how they appear in PACKAGES variable, so all these files will already be included in PN anyway and PN-tools won't be created because it's empty as you can see in packages-split directory:
$ find tmp-glibc/work/core2-64-oe-linux/geographiclib/1.48-r0/packages-split/geographiclib-tools/
tmp-glibc/work/core2-64-oe-linux/geographiclib/1.48-r0/packages-split/geographiclib-tools/
$

Either prepend it or use PACKAGE_BEFORE_PN variable.

I'll fix it.